nextflow_schema.json

```diff
@@ -479,6 +479,29 @@
                 }
             },
             "fa_icon": "fas fa-layer-group"
+        },
+        "deduplication": {
+            "title": "Deduplication",
+            "type": "object",
+            "description": "",
+            "default": "",
+            "properties": {
+                "skip_deduplication": {
+                    "type": "boolean",
+                    "description": "Specify to skip the removal of PCR duplicates."
+                },
+                "deduplication_tool": {
+                    "type": "string",
+                    "default": "markduplicates",
+                    "description": "Specify which tool to use for deduplication. Options: 'markduplicates', 'dedup'. ",
+                    "help_text": "Sets the duplicate read removal tool. By default uses `markduplicates` from Picard. Alternatively an ancient DNA specific read deduplication tool `dedup` (Peltzer et al. 2016) is offered. The latter utilises both ends of paired-end data to remove duplicates (i.e. true exact duplicates, as markduplicates will over-zealously deduplicate anything with the same starting position even if the ends are different). DeDup should only be used solely on paired-end data otherwise suboptimal deduplication can occur if applied to either single-end or a mix of single-end/paired-end data."
```

Comment:
```suggestion
                    "help_text": "Sets the duplicate read removal tool. By default uses `markduplicates` from Picard. Alternatively an ancient DNA specific read deduplication tool `dedup` (Peltzer et al. 2016) is offered. The latter utilises both ends of paired-end data to remove duplicates (i.e. true exact duplicates, as markduplicates will over-zealously deduplicate anything with the same starting position even if the ends are different). DeDup usage can only be used solely on paired-end data in the pipeline, as otherwise suboptimal deduplication can occur if applied to either single-end or a mix of single-end/paired-end data."
```